ee/session_recordings/session_summary/output_data.py

```diff
@@ -31,6 +32,15 @@ class EnrichedKeyActionSerializer(RawKeyActionSerializer):
     event_index = serializers.IntegerField(min_value=0, required=False, allow_null=True)
```

Comment:
To confirm, I would still send full window IDs and URLs to group summaries. Within the current scale, we would save not too many tokens to be worth it. Could iterate in the next PRs when scaling to hundreds.